### ISSUE TITLE
Add areas

### ISSUE THREAD

**Issue by algrw-c** [Mortennn/Dozer#141]
Is there any way to add more areas?
I mean, not the remove, add another dot that hiddes another zone or something like that

**Comment by afeblot** [comment]
Yes, that would allow to hide a section in the middle of the bar, while still keeping the left section where a new icon could have loaded while being hidden by the menu items.
This feature, used to reveal such a hidden new icon would be more efficient than hiding the menu items (in 4.2.0), which works but is not so elegant, with the active application blatantly switching to/from Dozer, with the application icon appearing/disappearing in the Dock.
I would have hoped that this was already possible with the 3rd Dozer dot, but sadly, option-clicking a dot hides not only the section between dot 3 and dot 2, but also between dot 2 and dot 1. I would have hoped it only hides the section between dot 2 and dot 1.
